docs/src/build.md

```diff
@@ -103,8 +103,13 @@ We provide two Docker images for tt-forge-fe:
 1. **Base Image**: This image includes all the necessary preinstalled dependencies.
 2. **Prebuilt Environment Image**: This image also comes with a prebuilt environment, allowing you to skip the environment build step.
 
-
-```
+```sh
 ghcr.io/tenstorrent/tt-forge-fe/tt-forge-fe-base-ird-ubuntu-22-04
 ghcr.io/tenstorrent/tt-forge-fe/tt-forge-fe-ird-ubuntu-22-04
 ```
+
+_Note: Before re-building Forge FE with new changes, make sure to update access rights FFE and MLIR toolchains:_
```

Comment:
Is this related only for docker images? I think we should be more clear, e.g.:

_Note: To be able to build `tt-forge-fe` inside the docker containers, make sure to set yourself as the owner of `tt-forge-fe` and `tt-mlir` toolchain directories:_

Reply:
Yup, only to docker images. 

There is already a section which covers these:

<img width="819" alt="image" src="https://github.com/user-attachments/assets/2415aeab-00fe-45bb-b321-1531574dd2c1">

However, being more explicit is a valid point :))